src/components/TopNav/hooks.ts

```diff
@@ -27,3 +28,18 @@ export function useNavigation() {
 
   return { currentPath, isActive };
 }
+export function findServiceByPath(servicePath: string): NavLink | null {
```

Comment:
It's weird to have this function in this file. Did you mean to put it in `config.ts`?